lunches.py

```diff
@@ -373,6 +373,23 @@ def rusty_bell_pub(dom):
         yield food
 
 
+@restaurant("Viktorka", "https://www.viktorkaostrava.cz/denni-menu/", Location.Poruba)
+def viktorka(dom):
+    sections = dom.css("ul.elementor-price-list")
+    type = [Soup, Lunch]
```

Comment:
`type` is reserved python name. Use `types`?

Reply:
done